rs/crypto/tree_hash/src/flat_map.rs

```diff
@@ -116,6 +116,17 @@ impl<K: Ord, V> FlatMap<K, V> {
             .ok()
     }
 
+    /// Returns references to the key/value pair with the largest key that is at most `key`.
+    ///
+    /// Complexity(O(log(N)))
```

Comment:
Personal preference:
```suggestion
    /// Complexity: `O(log(N))`
```
or just
```suggestion
    /// Complexity: O(log(N))
```